llm-token-limits/llm_token_limits.ipynb

```diff
@@ -0,0 +1,284 @@
+        "!pip install google-cloud-aiplatform"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "LpGncHDpentW"
+      },
+      "source": [
+        "## Authenticate your notebook environment (Colab only)\n",
+        "If you are running this notebook on Google Colab, run the following cell to authenticate your environment. This step is not required if you are using Vertex AI Workbench."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 1,
+      "metadata": {
+        "id": "8layJrBOZP4-"
+      },
+      "outputs": [],
+      "source": [
+        "import sys\n",
+        "\n",
+        "# Additional authentication is required for Google Colab\n",
+        "if \"google.colab\" in sys.modules:\n",
+        "    # Authenticate user to Google Cloud\n",
+        "    from google.colab import auth\n",
+        "\n",
+        "    auth.authenticate_user()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "-4eVYn8frc5i"
+      },
+      "source": [
+        "## Initialize notebook variables\n",
+        "\n",
+        "* **PROJECT_ID**: The default GCP project to use when making Vertex API calls.\n",
+        "* **REGION**: The default location to use when making API calls.\n",
+        "* **API_ENDPOINT**:  Desired API endpoint, e.g. https://apigee.iloveapimanagement.com/generate\n",
+        "* **API_KEY**: After deploying the sample you'll get 2 API keys: **Bronze Key** and **Silver Key**. First, set the value of your **Bronze Key**."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 5,
+      "metadata": {
+        "id": "kj-10KmnZSVe"
+      },
+      "outputs": [],
+      "source": [
+        "from langchain_google_vertexai import VertexAI\n",
+        "# Define project information\n",
+        "PROJECT_ID = \"\"  # @param {type:\"string\"}\n",
+        "LOCATION = \"\"  # @param {type:\"string\"}\n",
+        "API_ENDPOINT = \"https://REPLACE_WITH_APIGEE_HOST/v1/samples/llm-token-limits\"  # @param {type:\"string\"}\n",
+        "API_KEY = \"\"  # @param {type:\"string\"}\n",
+        "# Initialize Langchain\n",
+        "model = VertexAI(\n",
+        "      project=PROJECT_ID,\n",
+        "      location=LOCATION,\n",
+        "      api_endpoint=API_ENDPOINT,\n",
+        "      api_transport=\"rest\",\n",
+        "      streaming=True,\n",
+        "      model_name=\"gemini-1.5-pro\",\n",
+        "      additional_headers={\"x-apikey\": API_KEY})"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "S478piq-SGLc"
+      },
+      "source": [
+        "## Test tiered AI products\n",
+        "\n",
+        "Apigee allows you to create a tiered product strategy with different API access levels (e.g., Bronze, Bronze, Goals) to cater to diverse user needs and limits. During the [Setup](#setup) stage you deployed 2 AI Product tiers for testing purposes.\n",
```

Comment:
Was `(e.g., Bronze, Bronze, Goals)` supposed to be something like `(e.g., Bronze, Silver, Gold)`?